content/W2017/W17-event-beckwith-talk.md

```diff
@@ -0,0 +1,29 @@
+Title: Tech Talk: Hotspot Virtual Machine's Execution Engine by Monica Beckwith
```

Comment:
I like the idea of adding out of town, like in http://wics.uwaterloo.ca/2016/02/stephanie-morillo-talk/

`Out of Town Speaker: Hotspot Virtual Machine's Execution Engine by Monica Beckwith`